packages/@ember/octane-addon-blueprint/files/tests/dummy/config/environment.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(environment) {
       FEATURES: {
         // Here you can enable experimental features on an ember canary build
         // e.g. 'with-controller': true
```

Comment:
the comment should probably be updated as well

Reply:
https://github.com/ember-cli/ember-octane-blueprint/pull/35/commits/55005de0d26d9932170a47ab7b5ae42397e11133

Reply:
@NullVoxPopuli Done